### DISCUSSION TITLE
Are Custom CardElements supported in Teams Bots Framework?

### DISCUSSION THREAD

**Discussion by GlenPierce-Workday** [microsoft/AdaptiveCards#6914] [Q&A]
I am trying to follow the example found here: https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/javascript/extensibility on extending the adaptive card elements to create custom elements, but I'm receiving an error message in the console when I try to instantiate a card with one of my custom elements:

`[Bot invoke failed]: <BotError>Error when processing invoke response: Payload is incorrect`

Is Extensibility with the JavaScript SDK supported in Teams Bot Framework? If so, is there a specific version I should be using?

**Comment by tomlm** [comment]
The Teams client only supports a given version (1.4) of elements, and rejects payloads that don't match the schema. It's arguable that they shouldn't (as the intent was you ignore elements that are unknown).

The extensibility story only works if you are the host application yourself and you are targetting content to your own application.

**Reply by raviraushanweb** [reply]
Can you please elaborate a bit?
